wlm/src/main/java/ai/djl/serving/wlm/Adapter.java

```diff
@@ -53,6 +55,14 @@ public static Adapter newInstance(WorkerPoolConfig<?, ?> wpc, String name, Strin
             String modelName = wpc.getId();
             throw new IllegalArgumentException("The worker " + modelName + " is not a model");
         }
+
+        // TODO Allow URL support
+        try {
+            new URL(src);
```

Comment:
if we only want to support local path, we could just check if `src` is a local path?

Reply:
Better use URI, and check scheme is file or null, it more readable